### ISSUE TITLE
Timed messages provide no protection against malicious recipients

### ISSUE THREAD

**Issue by 0ki** [wireapp/wire-ios#392]
The number one reason why existing user base chose _wire_ over other communication tools is the expectation of higher security.

I empathise with the need to appeal to previously unclaimed audiences and timed messages is no doubt one of the ways to get there, however to minimise the impact on the high bar _wire_ has set for security it is crucial that every user, not just IT security specialists, are educated about the cosmetic nature of timed messages.

Therefore I ask that **when using the timed messages for the first time each user is shown a one-time alert** informing them of the security that timed messages can and cannot provide.

As a sample here's a CC0 licensed text you may use:
> WARNING! Timed messages are intended to be used only when you fully trust the parties receiving the message. They provide no protection against an adversary as there IS NO GUARANTEE that message will actually be removed on the other end, if the recipient is malicious.
>
> [I UNDERSTAND]

**Comment by raphaelrobert** [comment]
Thanks for that, this is an interesting proposal. We will take it into consideration!

**Comment by rbieb** [comment]
Just as an addition: I don't know how the situation is on ios, but android apps can prevent the user from taking screenshots when within a certain app, maybe this would be a good addition to wire, too.

**Comment by 0ki** [comment]
1) You would need to only block out the timed messages.
Taking screenshots of wire is a functionality I use daily.

2) This is futile as there are many clients available, also malicious user can compile their own iOS app version.

**Comment by IpsmLorem** [comment]
On the 2 : "blocking" at least express the will of the sender. 
The screenshoter may says to itself "Oh, right he doesn't want me to do that. I didn't know".

Of course, all client side security are just user guidance, targeted for non malicious users.

**Comment by lefherz** [comment]
I don't think you can prevent users from taking screenshots of the desktop or web client. At least there will always be the option to write a client which allows you to take screenshots of some area of the desktop.

So for 2 attack vectors (malicious recipient, LE registering a device) this means, you can't trust the cosmetic feature of timed messages.
